vmware/ngc/NGC-Register/src/main/conf/PluginVCenterConfig.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ROOT>
+	<VCENTER vcip="" vcname="" vcpwd="" hostip="" registerflag="0"/>
```

Comment:
Can we give some comments here about the flag 
like  1: already registered ; 0: not regisitered, so that user is aware about the configurable parameter.

Reply:
Fixed

Reply:
Thanks..

Reply:
Thanks..